src/main/java/modtrekt/model/module/UniqueModuleList.java

```diff
@@ -36,6 +36,33 @@ public boolean contains(Module toCheck) {
         return internalList.stream().anyMatch(toCheck::isSameModule);
     }
 
+    /**
+     * Returns true if the list contains an equivalent person as the given argument.
+     */
+    public boolean containsCode(ModCode toCheck) {
+        requireNonNull(toCheck);
+        ObservableList<Module> checkList = internalList.filtered(x -> x.getCode().equals(toCheck));
+
+        if (checkList.size() > 0) {
+            return true;
+        }
+        return false;
```

Comment:
These lines are equivalent to:
```suggestion
        return checkList.size() > 0;
```